connmgr/null.go

```diff
@@ -0,0 +1,20 @@
+package connmgr
+
+import (
+	"context"
+
+	"github.com/libp2p/go-libp2p-core/network"
+	"github.com/libp2p/go-libp2p-core/peer"
+)
+
+type NullConnMgr struct{}
```

Comment:
@raulk Out of curiosity, what is the purpose of the null implementation?

Reply:
It's used by the blank host: https://github.com/libp2p/go-libp2p-blankhost/blob/master/blank.go#L30.